library/Pdfexport/ProvidedHook/Pdfexport.php

```diff
@@ -66,7 +66,24 @@ public function htmlToPdf($html)
         // the object is destructed
         $chrome = $this->Chrome();
 
-        return $chrome->fromHtml($html)->toPdf();
+	$pdf = $chrome->fromHtml($html)->toPdf();
+
+	if ($html instanceof PrintableHtmlDocument && ($coverPage = $html->getCoverPage()) !== null) {
+            $coverPagePdf = $chrome
+                ->fromHtml((new PrintableHtmlDocument())
+                    ->add($coverPage)
+                    ->addAttributes($html->getAttributes())
+                    ->removeMargins()
+                )
+                ->toPdf();
+
+            $merger = new Merger(new TcpdiDriver());
+            $merger->addRaw($coverPagePdf);
+            $merger->addRaw($pdf);
+
+	    $pdf = $merger->merge();
+	}
+	return $pdf;
```

Comment:
Please add an empty line after closing braces

Reply:
Thanks, I just changed the code according to your suggestions.